doc/manipulating_visualizing/data_preparation.rst

```diff
@@ -244,11 +271,14 @@ computes automatically the translation part of the affine.
 Smoothing
 .........
 
-If smoothing the data prior to converting to voxel signals is required, it can
-be performed by :class:`NiftiMasker`. It is achieved by passing the full-width
-half maximum (in millimeter) along each axis in the parameter `smoothing_fwhm`.
+:class:`NiftiMasker` can further be used for local spatial filtering of
+the neuroimaging data to make the data more homogeneous and thus account
+for inter-individual differences in neuroanatomy and preprocessing.
```

Comment:
Referring to my previous note. If you use `preprocessing` here, people may be troubled because you call smoothing itself `preprocessing`.